### DISCUSSION TITLE
Electricity XXXX XXX Current Consumption seems too low?

### DISCUSSION THREAD

**Discussion by razorhazor** [BottlecapDave/HomeAssistant-OctopusEnergy#290] [General]
I have an Octopus home mini and have configured the Energy Dashboard in HACS.

I can see two entities that seem to relate to consumption:

'Electricity XXXX XXX Current Consumption' and 'Electricity XXXX XXX Current Demand'

Current Demand appears to match my Octopus real time demand (e.g. 1.52kW) and Current Consumption seem to be very low (0.026kWh).  

Am I misinterpreting the values? I would have thought Current Demand is the correct entity to use as it matches my smart meter display?

**Comment by BottlecapDave** [comment]
Hello.

As per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/sensors/electricity.md#current-consumption-electricity), this is not accumlative over the day. This is how the data is retrieved/returned by the APIs to best support the energy dashboard. It represents the spikes of data when looking at the live view within the app. 

If you want the current day's accumulative data, it's best used in conjunction with a utility meter (with `delta values` set to true). I didn't see the point in recreating something that is available within HA, however I might change this in the future due to confusion caused. However I'm trying to avoid this as it will be a breaking change.

**Reply by razorhazor** [reply]
Thanks for the reply.  I have configured the dashboard to use Current Consumption as per the docs and can see the following for today, which appears very low compared to usage in the Octopus App.

<img width="1247" alt="screenshot" src="https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/29519427/2ac79c05-01e9-4af0-b22f-dd3e77e835ab">

I had assumed this was because the totals were being calculated from the Current Consumption entity which was a very low number as opposed to real time current demand which appeared to match my smart meter and the octopus app dashboard.

**Reply by razorhazor** [reply]
my total energy usage from 10:00-10:30 was 1.44kWh and 10:30-11:00 2.32kWh

**Reply by BottlecapDave** [reply]
The energy dashboard does indeed accumulates the data. I was _slightly_ wrong in my previous statement in that it should match the data on the "day/today" view within the app. Was the home mini entity configured within HA at 10am? If it was, do you have any errors in your HA logs? If not, then that would be why you don't see the data, as the integration doesn't backport information (as HA doesn't support importing this without jumping through hoops like the previous consumption entity)
